### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by maxmohash**
Updated Readme to use the right command with some basic guideline for precaution.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,10 @@ Install
 ```bash
 $ brew install https://raw.githubusercontent.com/Reflejo/ib-unfuck-git/master/Formula/ib-unfuck-git.rb
 ```
+
+Usage
+-------
+
+Important: Make sure your changes are still in unstaged state only then this will remove the unwanted changes.
+
+Run `ibunfuck` command from the Lyft-iOS directory to remove the unwanted changes.
```
Comment (Reflejo) [inline README.md]: Can you replace `the Lyft-iOS directory` to `your repository directory`. This is not exclusively for lyft :)

Comment (maxmohash) [inline README.md]: Sure will do it

**Comment by Reflejo** [comment]
Thanks @maxmohash! I'll merge after you squash the changes

**Comment by maxmohash** [comment]
@Reflejo My Pleasure and the squashing is done!

**Pull request merged**